radio/src/Makefile

```diff
@@ -404,7 +404,11 @@ else
 endif
 TRANSLATE = -$(PYTHON) ../util/translate.py
 
-VERSION = 2.2.0$(OPENTX_VERSION_SUFFIX)
+VERSION_MAJOR    = 2
```

Comment:
I should not, the version will be parsed from stamp.h (like it is in nightly-21 already).